bot/core/bot.py

```diff
@@ -1,20 +1,61 @@
 from datetime import datetime
+from itertools import cycle
+import os
 
-from discord import Color, Embed
+import asyncpg
+from discord import Color, Embed, Game
+from discord.ext import tasks
 from discord.ext.commands import Bot as Base_Bot
 
 from bot import config
 
 
 class Bot(Base_Bot):
-    def __init__(self, extensions: list, *args, **kwargs) -> None:
+    """Subclassed Hotwired bot."""
+
+    def __init__(self, extensions: list, prefix: str, *args, **kwargs) -> None:
+        """Initialize the hotwired bot."""
         super().__init__(*args, **kwargs)
         self.extension_list = extensions
-        self.initial_call = True
+        self.first_on_ready = True
+        self.status = cycle(
+            [
+                (
+                    "😁Working At The Codin' Hole! Join me at "
+                    "https://discord.gg/aYF76yY"
+                ),
+                (
+                    "▶Check out My Creator's Youtube channel : "
+                    "https://www.youtube.com/channel/UC3S4lcSvaSIiT3uSRSi7uCQ/"
+                ),
+                f"Ping me using {prefix}help",
+                (
+                    "Official Instagram of My Creator ❌ "
+                    "https://instagram.com/the.codin.hole/"
+                ),
+                (
+                    "Ready To Work and Get Worked! My Github 🔆 "
+                    "https://github.com/janaSunrise"
+                ),
+            ]
+        )
+
+        self.pool = None
```

Comment:
`self.status` was removed quite some time ago after we decided to use a single status without changing it every now and then, there is no reason to re-implement it here.

Also doing so has nothing to do with this PR (add help system) it doesn't belong here.
Please remove this and use the `activity=Game(name=f"Ping me using {config.COMMAND_PREFIX}help"),` as it was before in `__main__.py`

Reply:
Okay. Anyway I prefered it this way, without all the cycle